docs/user-guide/knowledge_distillation.rst

```diff
@@ -0,0 +1,359 @@
+.. include:: /content/nemo.rsts
+
+SFT with Knowledge Distillation
+@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+
+Knowledge distillation is a technique in which a smaller (student) model learns from a larger (teacher) model. The goal is to "distill" information from the teacher to the student,
+resulting in a small model with comparable capabilities to the large model. Compared to standard SFT which trains the model to predict the next token,
```

Comment:
_'a small model with comparable capabilities to the large model'_ is not accurate, cuz we don't expect the student to be as good as the teacher.